### ISSUE TITLE
print versus println to stdout

### ISSUE THREAD

**Issue by andrew-saydjari** [JuliaPy/PythonCall.jl#290]
In transitioning from PyCall.jl, I was just trying to confirm that the stdout forwarding worked similarly and came across an example I found puzzling. 

```
@py begin
    import numpy as np
    a = np.zeros(10)
    println(a)
end
```

Completes and prints as normal

```
@py begin
    import numpy as np
    a = np.zeros(10)
    print(a)
end
```

Fails and gives the error "Python: TypeError: memoryview: a bytes-like object is required, not 'str'". 

I am just trying to reproduce the functionality I had before with py_str and using the `pyimport("sys")."stdout" = PyTextIO(stdout)`. My concern is that sometimes I want to import and run python packages that are doing print and not println and would like that to work without me having to edit them. Is this expected behavior or have I done something wrong/there is a better way to be going about this?

```
Julia Version 1.8.2
Commit 36034abf260 (2022-09-29 15:21 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
[6099a3de] PythonCall v0.9.12
```

**Comment by andrew-saydjari** [comment]
Yeah, I tried some variants of that. The suggestion you made works as you said. Thanks! But, I would be interested in finding a cleaner way as well. The only thing I am trying to do here is preserve the print statements internal to python packages that I am loading (that, without this hack, are not being printed below the code block that calls them). The forwarding of the python print statements to Julia stdout seems like it would be the expected (or at least desired) behavior.

**Comment by cjdoris** [comment]
Stuff printed by Python code to stdout/stderr should still appear no problem. Are you running in some other environment which redirects stdout, such as Jupyter?

**Comment by andrew-saydjari** [comment]
Right, I should have said this is in the Jupyter context. This leads to some odd behavior with the recommending fix I mentioned solving the problem for PyCall.
